### PR TITLE
Production Updates

### PR DESCRIPTION
**Pull request by janvig**

### CODE REVIEW

**Comment by sanket143** [comment]
Also update the script for build testing in `travis.yml`.

`npm run dist:snap`

**Comment by janvig** [comment]
> Also update the script for build testing in `travis.yml`.
> 
> `npm run dist:snap`

Done :+1:

File: .travis.yml [hunk 1]
```diff
@@ -5,3 +5,4 @@ node_js:
 script:
   - npm run lint
   - npm run dist
```
Comment (sanket143) [inline .travis.yml:7]: Also remove this script, cuz we don't have such rule in our package.json. Travis failed because of this.

Comment (sanket143) [reply .travis.yml:7]: `npm run dist`

Comment (janvig) [reply .travis.yml:7]: Oh my bad!

**Pull request merged**